spec/models/proxy_performance_spec.rb

```diff
@@ -4,7 +4,37 @@
   let(:site) {create(:site)}
   let(:proxy) {create(:proxy)}
   let(:proxy_performance) do
-    ProxyPerformance.create(:proxy => proxy, :site => site)
+    @proxy_performance ||= ProxyPerformance.create(
```

Comment:
Using a factory to generate `proxy_performance` wouldn't work with default arguments because I need it to use those specific site and proxy objects.  I'd still have to provide `:proxy` and `:site` as hash parameters.  If you think it looks cleaner, I could use the factory with those overrides.

I had to do the `||=` in order for the `creates a new ProxyPerformance object` test to work correctly.  Otherwise the after `:each` block would create a new `proxy_performance` object instead of re-use the one created in the test.  I could change the `let` into a `def` to make this more clear.